addons/project/static/tests/project_subtask_kanban_list_tests.js

```diff
@@ -72,13 +73,46 @@ QUnit.module('Subtask Kanban List tests', {
         await click(target, '.subtask_list_button');
 
         assert.containsOnce(target, '.subtask_list', "Clicking on the button should make the subtask list render, in this case we are expectig 1 list");
-        assert.containsN(target, '.subtask_list_row', 2, "The list rendered should show the open subtasks of the task, in this case 2");
-        assert.containsN(target, '.subtask_state_widget_col', 2, "Each of the list's rows should have 1 state widget, thus we are looking for 2 in total");
-        assert.containsN(target, '.subtask_user_widget_col', 2, "Each of the list's rows should have 1 user widgets, thus we are looking for 2 in total");
-        assert.containsN(target, '.subtask_name_col', 2, "Each of the list's rows should display the subtask's name, thus we are looking for 2 in total");
+        assert.containsN(target, '.subtask_list_row', 3, "The list rendered should show the open subtasks of the task, in this case 3");
+        assert.containsN(target, '.subtask_state_widget_col', 3, "Each of the list's rows should have 1 state widget, thus we are looking for 3 in total");
+        assert.containsN(target, '.subtask_user_widget_col', 3, "Each of the list's rows should have 1 user widgets, thus we are looking for 3 in total");
+        assert.containsN(target, '.subtask_name_col', 3, "Each of the list's rows should display the subtask's name, thus we are looking for 3 in total");
 
         await click(target, '.subtask_list_button');
 
         assert.containsNone(target, '.subtask_list', "If the drawdown button is clicked again, the subtasks list should be hidden again");
     });
+
+    QUnit.test("Update closed subtask count in the kanban card when the state of a subtask is set to Done.", async function (assert) {
+        let checkSteps = false;
+        const { openView } = await start({
+            serverData: { views: this.views },
+            mockRPC(route, { model, method, args }) {
+                if (checkSteps) {
+                    assert.step(`${model}/${method}`);
+                }
+            },
+        });
+        await openView({
+            res_model: "project.task",
+            views: [[false, "kanban"]],
+        });
+
+        assert.strictEqual(target.querySelector(".subtask_list_button").parentElement.textContent, "1/4");
+        checkSteps = true;
+        await click(target, '.subtask_list_button');
+        const subtaskEl = target.querySelector(".subtask_list")
```

Comment:
```suggestion
        const subtaskEl = target.querySelector(".subtask_list");
```
:stuck_out_tongue_closed_eyes: